modules/basic-auth/config/BASIC_AUTH.ts

```diff
@@ -0,0 +1 @@
+export default 'test:test'
```

Comment:
There shouldnt be a default value. Throw an error like in `modules/magento/config/MAGENTO_URL.ts`